lib/debugs_bunny/internal/schema/table_descriptor.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'column_descriptor'
+
+module DebugsBunny
+  module Internal
+    module Schema
+      class TableDescriptor
```

Comment:
This represents a database table and is used when generating the migration template